cfme/tests/webui/test_general_ui.py

```diff
@@ -567,6 +581,67 @@ def test_infrastructure_provider_left_panel_titles(
         soft_assert(test_view.is_displayed, f"{test_view} not displayed.")
 
 
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
```

Comment:
Needs to be ignored on upstream builds as well, since we're using an access.redhat.com URL.

Why only run against CFME 5.11?